python_modules/libraries/dagster-dbt/dagster_dbt/dbt_project.py

```diff
@@ -4,40 +4,41 @@
 from pathlib import Path
 from typing import List, Optional, Union
 
-from dagster._annotations import experimental, public
+from dagster._annotations import deprecated, experimental, public
 
 from .core.resources_v2 import DbtCliResource
 from .errors import DagsterDbtManifestNotPreparedError, DagsterDbtProjectNotFoundError
 
-logger = logging.getLogger("dagster-dbt.artifacts")
+logger = logging.getLogger("dagster-dbt.project")
 
 
 @experimental
-class DbtArtifacts:
+class DbtProject:
     def __init__(
         self,
         project_dir: Union[Path, str],
         *,
         target_folder: Union[Path, str] = "target",
-        prepare_command: List[str] = ["parse", "--quiet"],
+        create_manifest_command: List[str] = ["parse", "--quiet"],
```

Comment:
Would `prepare_project_command` would be an option? 

I think `create_manifest_command` and `create_manifest` make sense too. I'm just wondering if it'd be confusing for users because we recommend using `dbt parse` or `dbt compile` (for specific use cases) in our docs and Dag uni, both of which generate many more files than just `manifest.json`.

Reply:
maybe `generate_artifacts` / `generate_artifacts_command` ? In this context "artifacts" aligns with the dbt term accurately.

Reply:
I agree, "artifacts" would make a lot of sense.